openapi.yaml

```diff
@@ -8578,3 +8626,142 @@ components:
+          items:
+            type: string
+          type: array
+        jira:
+          $ref: '#/components/schemas/JIRAFilter'
+        environments:
+          description: Deployments must belong to one of these environments.
+          items:
+            type: string
+          type: array
+        conclusions:
+          description: Deployments must execute with any of these conclusions.
+          items:
+            $ref: '#/components/schemas/DeploymentConclusion'
+          type: array
+        with_labels:
+          description: Deployments should contain at least one of the specified label
+            values. `null` matches any label value and effectively checks the label
+            existence.
+          type: object
+        without_labels:
+          description: Deployments may not contain the specified label values. `null`
+            matches any label value and effectively ensures that the label does not
+            exist.
+          type: object
+      required:
+      - account
+      - date_from
+      - date_to
+      type: object
+    FilteredDeployments:
+      description: Found deployments, response from `/filter/deployments`.
+      items:
+        $ref: '#/components/schemas/FilteredDeployment'
+      type: array
+    DeploymentAnalysis:
+      additionalProperties: false
+      description: Result the analysis of the deployment.
+      properties:
+        code:
+          additionalProperties: false
+          description: Summary of the deployed code.
+          properties:
+            prs:
+              description: Number of deployed pull requests per repository.
+              additionalProperties:
+                minimum: 0
+                type: integer
+              type: object
+            lines_prs:
+              description: Number of changed lines in the deployed PRs per repository.
+              additionalProperties:
+                minimum: 0
+                type: integer
+              type: object
+            lines_overall:
+              description: Number of changed lines in the deployment per repository.
+              additionalProperties:
+                minimum: 0
+                type: integer
+              type: object
+            commits_prs:
+              description: Number of deployed PR commits per repository.
+              additionalProperties:
+                minimum: 0
+                type: integer
+              type: object
+            commits_overall:
+              description: Number of deployed commits per repository.
+              additionalProperties:
+                minimum: 0
+                type: integer
+              type: object
+          required:
+          - commits_overall
+          - commits_prs
+          - lines_overall
+          - lines_prs
+          - prs
+          type: object
+        releases:
+          allOf:
+          - description: Deployed releases.
```

Comment:
What will be returned in `releases` exactly?

Reply:
The format is the same as the response of `/filter/releases`. Including the released PRs even. Because I've got those precomputed already.

Reply:
Ok this is great to know!